.ci/docker-run.sh

```diff
@@ -6,7 +6,8 @@ set -ex
 cd .ci
 
 if [ "$INTEGRATION" == "true" ]; then
-    docker-compose up --exit-code-from logstash
+    # remove the `--attach logstash` if you want to see all logs including elasticsearch container logs
+    docker-compose up --exit-code-from logstash --attach logstash
```

Comment:
Some Travis CI jobs failed due to too many logs. In order to be 100% with the changes not breaking anything, limiting the logs excluding elasticsearch container logs.